drum-kit/script.js

```diff
@@ -0,0 +1,22 @@
+function dramKit(){
+const removeTransition = (event) =>{
+    if (event.propertyName !== 'transform') return;
+    event.target.classList.remove('play');
+}
+
+const keys = document.querySelectorAll('.key');
+keys.forEach( el=>{
+el.addEventListener('transitionend', removeTransition);  
+})
+
+const presKey = (e) => {
+ const audio = document.querySelector(`audio[data-key="${e.keyCode}"]`);
```

Comment:
keyCode is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)